src/main/resources/application.properties

```diff
@@ -15,6 +15,8 @@ spring.messages.basename=messages/messages
 
 # Actuator
 management.endpoints.web.exposure.include=*
+management.endpoint.env.show-values=always
```

Comment:
Why do we need this settings? They as far as I know do not affect the endpoints availability, only the values being displayed in response to `/env` endpoint call.